### PR TITLE
initial frontend commit

### LINKED ISSUE #1: Create frontend base application files
- [x] Create a new project
- [x] Remove template files
- [x] Install Cypress dependencies

### PR DESCRIPTION
**Pull request by arixpsy**
resolve #1

### CODE REVIEW

File: frontend/cypress.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "baseUrl": "http://localhost:3000",
+    "fixturesFolder": "cypress/fixtures",
+    "integrationFolder": "cypress/test/e2e",
+    "pluginsFile": "cypress/plugins/index.js",
+    "screenshotsFolder": "cypress/screenshots",
+    "supportFile": "cypress/support/index.js",
+    "videosFolder": "cypress/videos"
+}
```
Comment (bxtsang) [inline frontend/cypress.json]: newline character

File: frontend/cypress/plugins/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = function (on, config) {
+    // configure plugins here
+}
```
Comment (bxtsang) [inline frontend/cypress/plugins/index.js]: add newline char

File: frontend/layouts/error.vue [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<template>
+  <v-app dark>
+    <h1 v-if="error.statusCode === 404">
```
Comment (bxtsang) [inline frontend/layouts/error.vue:3]: just a qn, is error a global variable?

File: frontend/nuxt.config.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        dark: {
+          primary: colors.blue.darken2,
+          accent: colors.grey.darken3,
+          secondary: colors.amber.darken3,
+          info: colors.teal.lighten1,
+          warning: colors.amber.base,
+          error: colors.deepOrange.accent4,
+          success: colors.green.accent3
+        }
+      }
```
Comment (bxtsang) [inline frontend/nuxt.config.js:67]: should we take out themes and put it in a separate file?

File: frontend/pages/inspire.vue [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        class="mb-5"
+      >
+      <blockquote class="blockquote">
+        &#8220;First, solve the problem. Then, write the code.&#8221;
+        <footer>
+          <small>
+            <em>&mdash;John Johnson</em>
+          </small>
+        </footer>
+      </blockquote>
```
Comment (bxtsang) [inline frontend/pages/inspire.vue:16]: I am inspired

**Review by bxtsang — CHANGES_REQUESTED** [review]

File: frontend/layouts/error.vue (hunk 3 above)
Comment (arixpsy) [reply frontend/layouts/error.vue:3]: its a prop from the parent.
error.vue generated by nuxt.

File: frontend/nuxt.config.js (hunk 4 above)
Comment (arixpsy) [reply frontend/nuxt.config.js:67]: its generated by installing vuetify. This is already a config file so no point separating. overwrite of themes however will be done in a separate sass file.

**Pull request merged**